requirements_test.txt

```
@@ -8,5 +8,6 @@ mockcache==1.0.3; python_version < '3.0'
 pymemcache==3.2.0; python_version >= '3.0'
 ptvsd==4.3.2
 pytest==4.6.9; python_version < '3.0'
+mock==3.0.5
```

```suggestion
mock==3.0.5; python_version < '3.5'
```
This module is available as `unittest.mock` in currently supported versions of CPython.